crypto/signing/key.go

```diff
@@ -0,0 +1,159 @@
+package signing
```

Comment:
We need to add a TODO here to remove this package when we will have elrond-go-core

Reply:
done